docs/API.REST.md

```diff
@@ -697,3 +699,122 @@ Bulk import only works on documents in our persistent data storage layer.
   }
 }
 ```
+
+---
+
+### Getting the last statistics frame
+
+Kuzzle monitors its internal activities and make snapshots of them. This command allows getting the last stored statistics frame.
+
+These statistics include:
+
+* the number of connected users for protocols allowing this notion (websocket, udp, ...)
+* the number of ongoing requests
+* the number of completed requests since the last frame
+* the number of failed requests since the last frame
+
+**URL:** ``http://kuzzle:7512/api/_getStats``
+
+**Method:** ``GET``
+
+**Response:**
+
+```javascript
+{
+  error: null,                      // Assuming everything went well
+  result: {
+    count: <number of found documents>
+    _source: {                      // Your original count query
+      ...
+    },
+    collection: '<data collection>',
+    action: 'getStats',
+    controller: 'admin',
+    statistics: {
+      "YYYY-MM-DDTHH:mm:ss.mmmZ": {
```

Comment:
One last possibility would be to leave the response for ``getAllStats`` as it is, and to reformat the one for ``getStats`` like this:

```
{
  error: null,
  result: {
    ...
    statistics: {
      timestamp: "YYYY-MM-DDTHH:mm:ss.mmm",
      ... statistics informations ...
    }
}
```

This would allow reading the result of this request without resorting to ``Object.keys``, as it seems a bit superfluous for only one key.